src/corehost/common/longfile.windows.cpp

```diff
@@ -65,6 +69,19 @@ bool AssertRepeatingDirSeparator(const pal::string_t& path)
 
     assert(path_to_check.find(altDirSeparator) == pal::string_t::npos);
 
+    pal::string_t combDirSeparator1;
+    combDirSeparator1.push_back(LongFile::DirectorySeparatorChar);
+    combDirSeparator1.push_back(LongFile::AltDirectorySeparatorChar);
+
+    assert(path_to_check.find(combDirSeparator1) == pal::string_t::npos);
```

Comment:
Is this an invariant that will always hold? Can there be a case if this happen because if it does, then the assert will not help.

Same comment about the change below.